extensions/pocket/package.json

```
@@ -63,7 +85,7 @@
     }
   ],
   "dependencies": {
-    "@raycast/api": "^1.27.1"
+    "@raycast/api": "^1.30.3"
```

```suggestion
    "@raycast/api": "^1.31.0"
```
If you want the newest 😄